### PR TITLE
Env: non-empty GOBIN & GOPATH

### PR DESCRIPTION
**Pull request by Loyalsoldier**
Return non-empty GOBIN & GOPATH environment variable if not set explicitly by user.

### CODE REVIEW

**Comment by flycash** [comment]
CI failed， please check it.

**Comment by Loyalsoldier** [comment]
Wait for https://github.com/beego/beego/pull/4612 to be merged first.

**Comment by Loyalsoldier** [comment]
CI failed without enough error information. Any idea?

**Comment by flycash** [comment]
We don't care about the order of changes in the CHANGELOG.md. So you could choose any line of CHANGELOG.md you want to write the change. It could reduce the conflicts

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # developing
+
+- Env: non-empty GOBIN & GOPATH. [4613](https://github.com/beego/beego/pull/4613)
```
Comment (flycash) [inline CHANGELOG.md:3]: you could put this anywhere so that you won't get conflicts.

Comment (Loyalsoldier) [reply CHANGELOG.md:5]: Got it.

**Comment by Loyalsoldier** [comment]
Done.

**Review by flycash — APPROVED** [review]

**Pull request merged**